### PR TITLE
 Indico tutorial improvements

### PR DESCRIPTION
**Pull request by javierdelapuente**
Applicable spec: <link>

### Overview

Rename the tutorial title, from "Quick guide" -> "Deploy the Indico charm for the first time".
Added creating and destroy the model in the tutorial.

Also minor fixes:
Use redis in the channel latest/edge, as the stable channel has not been updated since 2021.
Fix typo in command, that was using app name `postgresql` instead of `postgresql-k8s`.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -11,11 +11,20 @@ Through the process, you'll inspect the Kubernetes resources created, verify the
...
-- Juju controller and model created.
+- Juju microk8s controller created and active.
 - NGINX Ingress Controller. If you're using [MicroK8s](https://microk8s.io/), this can be done by running the command `microk8s enable ingress`. For more details, see [Addon: Ingress](https://microk8s.io/docs/addon-ingress).
 
 For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
 
+### Add a Juju model for the tutorial
+
+To manage resources effectively and to separate this tutorial's workload from
+your usual work, we recommend creating a new model using the following command.
```
Comment (erinecon) [inline docs/tutorial.md:22]: Nitpick: "we recommend creating a new model using the following command." --> "create a new model using the following command."

File: docs/tutorial.md [hunk 2]
```diff
@@ -141,4 +150,14 @@ The default hostname for the Indico application is `indico.local`. To resolve it
 
 Optional: run `echo "127.0.0.1 indico.local" >> /etc/hosts` to redirect the output of the command `echo` to the end of the file `/etc/hosts`.
 
-After that, visit `http://indico.local` in a browser and you'll be presented with a screen to create an initial admin account.
\ No newline at end of file
+After that, visit `http://indico.local` in a browser and you'll be presented with a screen to create an initial admin account.
+
+
+## Cleaning up the Environment
```
Comment (erinecon) [inline docs/tutorial.md:156]: Nitpick: Cleaning --> Clean

**Comment by erinecon** [comment]
Looks great! Only minor nitpicks from me. Otherwise I approve of these changes :+1:

**Review by erinecon — DISMISSED** [review]
Looks great!

**Review by arturo-seijas — DISMISSED** [review]

**Review by amandahla — DISMISSED** [review]
LGTM

**Review by erinecon — APPROVED** [review]

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**